### PR TITLE
Seats

### PR DESCRIPTION
**Pull request by aviau**
Based on #69

Implémentation des seats dans l'API.

### CODE REVIEW

File: backend/api/internal/handlers/floorplans/seats/handlers.go [hunk 1]
```diff
@@ -15,23 +15,9 @@ import (
...
-			w.WriteHeader(http.StatusInternalServerError)
-			return
-		}
-
-		if floorplan == nil {
-			w.WriteHeader(http.StatusNotFound)
-			return
-		}
-		// ENDS HERE
+		floorplan := decorators.FloorplanFromContext(r.Context())
```
Comment (zergov) [inline backend/api/internal/handlers/floorplans/seats/handlers.go]: waaaaaaaaaaw <3

File: backend/api/internal/handlers/floorplans/handlers.go [hunk 2]
```diff
@@ -62,24 +59,12 @@ func floorplansPostHandler(app *app.App) http.Handler {
...
-			w.WriteHeader(http.StatusInternalServerError)
-			return
-		}
-
-		if floorplan == nil {
-			w.WriteHeader(http.StatusNotFound)
-			return
-		}
-
+		floorplan := decorators.FloorplanFromContext(r.Context())
```
Comment (zergov) [inline backend/api/internal/handlers/floorplans/handlers.go]: Double waaaaaaaaaaaaaw <3

File: backend/api/internal/handlers/floorplans/seats/handlers.go [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+	}
+
+	handler = decorators.FloorplanContext(app, handler)
+	handler = decorators.JsonHeaders(handler)
+
+	return http.HandlerFunc(handler)
+}
+
+func seatsPostHandler(app *app.App) http.Handler {
```
Comment (zergov) [inline backend/api/internal/handlers/floorplans/seats/handlers.go]: I think we should think about the use case for this call. Don't you think we should have an endpoint to add a list of Seats ? 

IMO the common use case is going to be the addition of a "row" of seats. Now, the "row" itself is abstract and could be something else, but most likely the addition of seats in a floorplan is going to be something else than a single call by seat. 

A floorplan of 3000 seats is a lot of API calls. I think this endpoint should exist, just so that we're following the rest format. But an other endpoint is going to be needed in the future.

TLDR: Jpense qu'on va avoir besoin d'un call qui permet d'ajouter une liste de seats.

File: backend/api/internal/handlers/floorplans/seats/handlers_test.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
+package seats_test
```
Comment (zergov) [inline backend/api/internal/handlers/floorplans/seats/handlers_test.go]: Pleins de beau tests <3
